spec/index.bs

```diff
@@ -1294,11 +1294,7 @@ At the beginning of step 5, insert the following steps:
 Note: See <a href="https://github.com/tc39/ecma262/issues/938">TC39/ecma262 issue #938</a>
 (adding a string to be compiled to algorithm parameters)
 
-Issue: Should we be modifying
-[[HTML5#hostensurecancompilestrings(callerrealm,-calleerealm)|HostEnsureCanCompileStrings]]
-or EnsureCSPDoesNotBlockStringCompilation?
-
-Modify [[CSP#can-compile-strings|HostEnsureCanCompileStrings]]
+Modify [[HTML5/webappapis#section-hostensurecancompilestrings|HostEnsureCanCompileStrings]]
```

Comment:
IIRC, this HTML abstract-op just calls through to the CSP one, so it seems less likely to cause problems if the change is made deeper.

Reply:
The issue is that I want to have less integration points (to later sync the amendment to other specs), and I already integrate with HTML, so let's keep it like this.